client/js/store/map/Layers.js

```diff
@@ -224,8 +224,17 @@ const LayersStore = {
             'layers',
             'layerType',
             'visibilityGroupId',
-            'isMinimap'
+            'isMinimap',
+            'categoryId'
           ].join()
+        },
+        filter: {
```

Comment:
Why this filter?

Reply:
because a procedure can have several gisLayerCategories. (in some projects). We have to fetch exactly the root-category. And thats the one with no parentId.
With that we get the other Categories as relationships in the included-Section, which reflects the old behavior and makes the changes easier. (if not the whole dattastructure)